### ISSUE TITLE
Dismiss button on Revocation Notification in Credential Details does not have an Accessibility Label or TestID

### ISSUE THREAD

**Issue by nodlesh** [bcgov/bc-wallet-mobile#950]
### Steps to Reproduce
``` gherkin
Given I have a credential
And the Issuer has revoked the credential
When I select to view the revocation notification
Then the Dismiss button on the Credential Details Revocation Notification is not accessible or have a TestID


### Expected Result
There would be the standard `com.ariesbifold:id/Dismiss` testID, and a `Dismiss` accessibility label. 

 
### Environment
BC Wallet build 730
Tested on iOS 15 & 16

### Workaround
None known

### Severity
Medium

### Priority
High

### Screenshots or Video
![Screenshot 2023-03-07 at 11.08.40 AM.png](https://images.zenhubusercontent.com/5e3864b3b64bc2429a017886/a362e930-be49-4de9-ad39-f7f1a0d2657a)

### Test Impact
T011.1-Proof, T011.2-Proof

**Comment by jleach** [comment]
Fixed

![Screenshot 2023-05-25 at 1.33.50 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/335ebaed-7ab1-4816-a227-ee7fd1be7948)
